CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [unreleased]
+### Added
+- #1080 Checking for write errors caused by not enough disk space
+
+### Changed
```

Comment:
No necessary to add code changes in the changelog since people looking at it care only about app functionality changes.